### PR TITLE
high pri Jit builtins

### PR DESCRIPTION
**Pull request by ailzhang**
bin/hex/oct/round/chr

### CODE REVIEW

File: aten/src/ATen/core/interned_strings.h [hunk 1]
```diff
@@ -135,6 +135,10 @@ namespace c10 {
   _(aten, wait)                    \
   _(aten, save)                    \
   _(aten, ord)                     \
+  _(aten, chr)                     \
```
Comment (zdevito) [inline aten/src/ATen/core/interned_strings.h]: I am not sure these values are actually used. I only see them as strings not a symbols.

**Review by zdevito — APPROVED** [review]

File: aten/src/ATen/core/interned_strings.h (hunk 1 above)
Comment (ailzhang) [reply aten/src/ATen/core/interned_strings.h]: Yea somehow it didn't compile without adding them here. I will double check before merging. Thanks!

File: torch/csrc/jit/script/compiler.cpp [hunk 2]
```diff
@@ -429,6 +429,12 @@ struct Environment {
           {"all", std::make_shared<BuiltinFunction>(aten::all, at::nullopt)},
           {"list", std::make_shared<BuiltinFunction>(aten::list, at::nullopt)},
           {"ord", std::make_shared<BuiltinFunction>(aten::ord, at::nullopt)},
+          {"chr", std::make_shared<BuiltinFunction>(aten::chr, at::nullopt)},
+          {"hex", std::make_shared<BuiltinFunction>(aten::hex, at::nullopt)},
```
Comment (eellison) [inline torch/csrc/jit/script/compiler.cpp]: `round` `hex` and `oct` are magic methods, and round has a second argument for number of decimal places that defaults to 0

https://rszalski.github.io/magicmethods/

Comment (ailzhang) [reply torch/csrc/jit/script/compiler.cpp]: @eellison I noticed the second optional argument for `round` but it has a weird behavior that is hard to match from cpp side. See notes in https://docs.python.org/3/library/functions.html#round. 
Also I think round is rarely with that arguments, would prefer to delay implementing it unless it's necessary. :P

File: aten/src/ATen/core/interned_strings.h (hunk 1 above)
Comment (ailzhang) [reply aten/src/ATen/core/interned_strings.h]: These are used as symbols in `compiler.cpp`, so it makes sense to add them here I think.

File: torch/csrc/jit/script/compiler.cpp (hunk 2 above)
Comment (ailzhang) [reply torch/csrc/jit/script/compiler.cpp]: `round` with optional args's impl in cpython is a bit long. I will make that a low pri item for following patches.

Comment (eellison) [reply torch/csrc/jit/script/compiler.cpp]: `round` without arguments is the same thing as `int`, so there's not much utility in having it without the argument.

Comment (ailzhang) [reply torch/csrc/jit/script/compiler.cpp]: @eellison They're different :P 
```
round(2.6)=3
int(2.6)=2
```

**Pull request closed without merging**